components/eventing-controller/testing/subscriber.go

```diff
@@ -111,6 +132,40 @@ func (s Subscriber) CheckEvent(expectedData, subscriberCheckURL string) error {
 	return nil
 }
 
+func (s Subscriber) CheckRetries(expectedData int, subscriberCheckRetriesURL string) error {
```

Comment:
please add a comment describing the functionality

Reply:
done